src/main/java/com/example/waytogo/route/controller/RouteController.java

```diff
@@ -0,0 +1,77 @@
+package com.example.waytogo.route.controller;
+
+import com.example.waytogo.route.mapper.RouteMapper;
+import com.example.waytogo.route.model.dto.GetRouteResponse;
+import com.example.waytogo.route.model.dto.PostRouteRequest;
+import com.example.waytogo.route.model.dto.PutRouteRequest;
+import com.example.waytogo.route.model.dto.RouteDTO;
+import com.example.waytogo.route.model.entity.Route;
+import com.example.waytogo.route.service.impl.RouteServiceJPA;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@RequiredArgsConstructor
+@RestController
+public class RouteController {
+    public final String ROUTE_PATH = "api/routes";
+    public final String ROUTE_PATH_ID= ROUTE_PATH + "/{routeID}";
+    private final RouteServiceJPA routeService;
+    private final RouteMapper routeMapper;
+
+    @GetMapping(ROUTE_PATH)
+    public ResponseEntity getRoutes() {
+        return new ResponseEntity<>(null, HttpStatus.OK);
+    }
+
```

Comment:
ogolnie nie bede ci wszedzie komentowal, ale prosze mi tu wypierdalac tak jak pisalismy z tym Route w kontrolerze, wiec tam nizej nie bede komentowal o tym ale tez popraw, serwis zwraca DTO xD, fajnie jakbys tez to zmienil jak masz optionala zebys streamami porobil bo to mniej czytelne co masz(zobacz jak na labach mialem np )

a no i tak jak mowilismy GETRouteResponse -nie